honcho/export/base.py

```diff
@@ -19,17 +20,22 @@
 
 
 class BaseExport(object):
-    def __init__(self, template_env=None):
+    # Override this in subclasses -- e.g.: `supervisord`, `upstart`, etc.
+    default_template_dir = ''
+
+    def __init__(self, template_dir=None, template_env=None):
         if template_env is None:
             template_env = _default_template_env()
         self._template_env = template_env
+        self._template_dir = template_dir or self.default_template_dir
```

Comment:
I think I'd rather you made the `_template_dir` option `None` by default, and only changed the `jinja2` loader to a `FileSystemLoader` when `self._template_dir is not None`.